source/fundamentals/connection.txt

```diff
@@ -6,14 +6,14 @@ Connection Guide
 
 .. toctree::
 
-   /fundamentals/connection/connect
-   /fundamentals/connection/connection-options
-   /fundamentals/connection/mongoclientsettings
-   /fundamentals/connection/network-compression
-   /fundamentals/connection/tls
-   /fundamentals/connection/socks
-   /fundamentals/connection/jndi
-   Connect to MongoDB Atlas from AWS Lambda <https://www.mongodb.com/docs/atlas/manage-connections-aws-lambda/>
+   Connect to MongoDB </fundamentals/connection/connect>
+   Connection Options </fundamentals/connection/connection-options>
+   MongoClient Settings </fundamentals/connection/mongoclientsettings>
+   Network Compression </fundamentals/connection/network-compression>
+   TLS/SSL </fundamentals/connection/tls>
+   SOCKS5 Proxy Connection </fundamentals/connection/socks>
+   JNDI Datasource Connection </fundamentals/connection/jndi>
+   Connect with AWS Lambda <https://www.mongodb.com/docs/atlas/manage-connections-aws-lambda/>
```

Comment:
Since this is in the connection guide, it seems reasonable to call this "AWS Lambda"